src/annotator/test/annotation-sync-test.js

```diff
@@ -136,5 +136,24 @@ describe('AnnotationSync', function() {
         });
       });
     });
+
+    context('if the parent context has a crashing Function.bind polyfill', function() {
```

Comment:
I've left this test (and the one for sidebar below) here as I can't think of a better way. This obviously isn't ideal as every module that uses clean-context would need to have its own "doesn't crash" test.

I looked into replace `Function.prototype.bind` globally with a crashing function in the tests, but don't see any way to do that for all non-sidebar tests but _not_ do it for the `src/sidebar` tests.

I looked into writing an eslint rule to warn on uses of bind, but it seemed difficult to warn when `bind()` or `bind.call()` is called but _not_ when the calling code is in `src/sidebar` and _not_ when `cleanContext.bind.call()` is called. (It may be doable, I've not written eslint rules before, but there wasn't an obvious good way to do it.)

The remaining option would be to delete these tests, but that would simply mean that the actual bug fixes in this pull request would not be covered by tests. I don't think the presence of these tests is doing any particular harm or means that we'd necessarily have to add similar tests for every other piece of code that uses clean-context.